dev/DynamicDependency/MddDetourPackageGraph.cpp

```diff
@@ -44,28 +44,33 @@ HRESULT MddDetourPackageGraphInitialize()
     }
 
     // Detour package graph APIs to our implementation
-    DetourRestoreAfterWith();
-    DetourTransactionBegin();
-    DetourUpdateThread(GetCurrentThread());
-    DetourAttach(&(PVOID&)TrueGetCurrentPackageInfo, DynamicGetCurrentPackageInfo);
-    DetourTransactionCommit();
-
+    RETURN_IF_WIN32_BOOL_FALSE(DetourRestoreAfterWith());
```

Comment:
Does our code function if these things return false? How do we expect callers to recover / handle a failing hresult versus us doing a failfast and having some semblance of failure context? (I would hope we can lean towards failfast on developer/unexpected error)

Reply:
All their examples show this. No error handling. Grrrr....

Updating to FailFast if any of this chokes. What they do is pretty unlikely to fail, and if it does we're well into weird territory so not much caller can reasonably do about it. FailFast seems the best fit. We can always change it later if we get feedback needing more nuanced behavior